### PR TITLE
feat: Add Provider children prop type

### PR DESCRIPTION
**Pull request by JulianRojasBet**
### Summary

Add Provider component children prop type to make it works on React 18

### CODE REVIEW

**Review by devdumpling — APPROVED** [review]

File: src/createSynaptik.tsx [hunk 1]
```diff
@@ -17,6 +18,10 @@ export type StoreMap<T extends ConstructorMap> = {
   [P in keyof T]: InstanceType<T[P]>;
 };
 
+export type ProviderProps = {
+  children?: ReactNode;
```
Comment (alextranwork) [inline src/createSynaptik.tsx:22]: Should this be `ReactNode | ReactNode[] | null` ?

**Review by alextranwork — APPROVED** [review]
1 comment but not a blocker

File: src/createSynaptik.tsx (hunk 1 above)
Comment (JulianRojasBet) [reply src/createSynaptik.tsx:22]: Not sure but [seeing this type definition from @types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L230) I would say `ReactNode` should be enough both others are already included on it

**Comment by JulianRojasBet** [comment]
Is the canary package released as `0.0.0-16aacbb`?

**Pull request merged**